hokusai/production.yml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: metric-web
+spec:
+  replicas: 2
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: metric
+        component: web
+        layer: application
+      name: metric-web
+      namespace: default
+    spec:
+      containers:
+      - name: metric-web
+        env:
+        - name: NODE_ENV
+          value: production
+        - name: PORT
+          value: 8080
+        - name: STATSD_HOST
+          value: datadog
+        - name: STATSD_PORT
+          value: 8125
+        envFrom:
+        - configMapRef:
+            name: metric-environment
+        image: 585031190124.dkr.ecr.us-east-1.amazonaws.com/metric:production
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: metric
+    component: web
+    layer: application
+  name: metric-web
+  namespace: default
```

Comment:
you can add https support like this
https://github.com/artsy/volt/pull/2989/files#diff-9af8dd47d98e3e7912dadcf50c599d06R48